src/libcollections/btree/node.rs

```diff
@@ -1453,25 +1333,27 @@ unsafe fn move_edges<K, V>(
     dest.correct_childrens_parent_links(dest_offset, dest_offset + count);
 }
 
-impl<BorrowType, K, V, HandleType>
-        Handle<NodeRef<BorrowType, K, V, marker::LeafOrInternal>, HandleType> {
-
+impl<BorrowType, K, V, HandleType> Handle<NodeRef<BorrowType, K, V, marker::LeafOrInternal>,
+                                          HandleType> {
     /// Check whether the underlying node is an `Internal` node or a `Leaf` node.
-    pub fn force(self) -> ForceResult<
-        Handle<NodeRef<BorrowType, K, V, marker::Leaf>, HandleType>,
-        Handle<NodeRef<BorrowType, K, V, marker::Internal>, HandleType>
-    > {
+    pub fn force(self)
+                 -> ForceResult<Handle<NodeRef<BorrowType, K, V, marker::Leaf>, HandleType>,
+                                Handle<NodeRef<BorrowType, K, V, marker::Internal>, HandleType>> {
         match self.node.force() {
-            ForceResult::Leaf(node) => ForceResult::Leaf(Handle {
-                node: node,
-                idx: self.idx,
-                _marker: PhantomData
-            }),
-            ForceResult::Internal(node) => ForceResult::Internal(Handle {
-                node: node,
-                idx: self.idx,
-                _marker: PhantomData
-            })
+            ForceResult::Leaf(node) => {
+                ForceResult::Leaf(Handle {
+                    node: node,
+                    idx: self.idx,
+                    _marker: PhantomData,
+                })
+            }
+            ForceResult::Internal(node) => {
+                ForceResult::Internal(Handle {
+                    node: node,
+                    idx: self.idx,
+                    _marker: PhantomData,
+                })
+            }
```

Comment:
This is again mostly personal style and very debatable, but I don't see the benefit of this change.